client/my-sites/plan-features/index.jsx

```diff
@@ -936,7 +949,7 @@ const ConnectedPlanFeatures = connect(
 		const canPurchase = ! isPaid || isCurrentUserCurrentPlanOwner( state, selectedSiteId );
 		const isLoggedInMonthlyPricing =
 			! isInSignup && ! isJetpack && kindOfPlanTypeSelector === 'interval';
-		const flowName = getCurrentFlowName( state );
+		const flowName = ownProps.flowName ? ownProps.flowName : getCurrentFlowName( state );
```

Comment:
`flowName` is a stepper framework concept. Marketplace code should not touch/use `flowName` at all.

I don't actually know why `flowName` is even in here, probably shouldn't be.

Reply:
I came to realize that after digging into some more of the code today. Originally didn't want to introduce a new prop `isInMarketplace` and wanted to reuse a prop that already existed.

If we want to revert this, we'll also need to check on the code [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plan-features-comparison/index.jsx#L457). I remember that was the reason why I introduced a fake flowName at my [first attempt](https://github.com/Automattic/wp-calypso/pull/67891/files#diff-d89d039d749332c2d6c1c257980ee3273b0db6689a013b93a7abf58a9e6267f1R69).

Reply:
Going back to isInMarketplace instead of using the flowName sounds good and cleaner to me!